### DISCUSSION TITLE
系统监视器bug：磁盘读写详细信息无数据

### DISCUSSION THREAD

**Discussion by Aninana** [linuxdeepin/developer-center#4268] [Troubleshooting | 故障排查]
原帖子在：https://bbs.deepin.org/post/255893 ，而且不止我一个人发生过此情况。
打开左下角的磁盘信息时，右面的显示却无任何数据，一直为0。（左下角却有读写数据）
而且无论是nvme还是SATA磁盘都不行。
尝试使用终端sudo apt purge, sudo apt install 卸载重装过系统监视器，结果还是一样。

**Comment by BLumia** [comment]
20.9？是的话，23 有这个问题吗？

**Reply by Aninana** [reply]
是20.9，23没有试过

**Reply by Aninana** [reply]
刚刚在V23 Alpha2试了一下，结果也是这样
![截图_VirtualBox Machine_20230504125838](https://user-images.githubusercontent.com/130546027/236116201-bdc9b067-86db-4526-9341-db6a61127f30.png)

**Reply by BLumia** [reply]
@babyfengfjx 看看能否复现？
